test/mongo/mongo-index-policy.spec.ts

```diff
@@ -0,0 +1,147 @@
+import "expect-puppeteer";
+import { Frame } from "puppeteer";
+import { getTestExplorerFrame } from "../testExplorer/TestExplorerUtils";
+import { generateUniqueName } from "../utils/shared";
+
+const LOADING_STATE_DELAY = 1000;
+const RETRY_DELAY = 5000;
+const CREATE_DELAY = 10000;
+jest.setTimeout(300000);
+
+describe("MongoDB Index policy tests", () => {
+  it("Open, Create and Saved Index", async () => {
+    try {
+      const singleField_Id = generateUniqueName("key");
+      const wild_Id = generateUniqueName("key") + "$" + "**";
+      const frame = await getTestExplorerFrame();
+
+      //open dataBaseMenu
+      await frame.waitForSelector('div[class="splashScreen"] > div[class="title"]', { visible: true });
+      await frame.waitFor(LOADING_STATE_DELAY);
+      await frame.waitForSelector('div[class="splashScreen"] > div[class="title"]', { visible: true });
+      const databases = await frame.$$(`div[class="databaseHeader main1 nodeItem "] > div[class="treeNodeHeader "]`);
+      if(databases.length == 0) {
+        createDatabase(frame)
+      }
+      const selectedDbId = await frame.evaluate((element) => {
+        return element.attributes["data-test"].textContent;
+      }, databases[0]);
+
+      // Click into database
+      await frame.waitFor(`div[data-test="${selectedDbId}"]`);
+      await frame.waitFor(LOADING_STATE_DELAY);
+      await frame.click(`div[data-test="${selectedDbId}"]`);
+
+      //click intp scale& setting
```

Comment:
typo on "into",
maybe comment it as "click on" everywhere

Reply:
Sure, will changed